### DISCUSSION TITLE
Finding underpacked residues in a structure

### DISCUSSION THREAD

**Discussion by EDC145** [RosettaCommons/rosetta#114] [RosettaScripts]
Hi everyone
I am trying to find the residues in a protein which may be underpacked. How can I proceed with this? 
Thank you

**Comment by roccomoretti** [comment]
One approach to find "underpacked" residues would be to simply run the PackRotamersMover and look for residues which have changed significantly. You could find "changed" either by rmsd differences, or by looking at the energies. (Note, though, that energies of surrounding residues will also likely change, even if they didn't move. Whether this still counts as "underpacked" is something you'll have to consider for your particular use case.)

If you don't want to repack, another approach would be to take a look at the per-residue energies. (Either in the table in the output PDB, or by using the `per_residue_energies` application.) Depending on what you mean by "underpacked", you'll may find residues with too-low `fa_atr` score fit your criteria. (As those are the residues whose LJ attractive interactions/VdW contacts are "underperforming".)

All of this is contingent on what you mean by "underpacked" -- different definitions of "underpacked" are going to require slightly different approaches and different analysis schemes (cutoffs, etc.). It's worth thinking in a bit more detail about what it is you're looking for and trying to achieve. That should help in narrowing down the approach you're after.

**Comment by roccomoretti** [comment]
Marking this as "closed", as I hope your question has been answered. If that assessment is incorrect, feel free to re-open or open up a new discussion with more details.
